src/modules/lighthouse/auditUrl.ts

```diff
@@ -0,0 +1,38 @@
+import ora from 'ora'
+import chalk from 'chalk'
+
+import log from '../../logger'
+import { lighthouse, workspaces } from '../../clients'
+import { getAccount, getWorkspace } from '../../conf'
+import { TableGenerator } from './TableGenerator'
+
+const [account, currentWorkspace] = [getAccount(), getWorkspace()]
```

Comment:
Prefer to get this data inside the main function

Reply:
Got it! Changing this to get this data with the `SessionManager` as well.